platform-hub-api/app/controllers/support_requests_controller.rb

```diff
@@ -0,0 +1,37 @@
+class SupportRequestsController < ApiJsonController
+
+  include AgentsInitializer
+
+  skip_authorization_check :only => [ :create ]  # Any authenticated user can create a support request
+
+  # POST /support_requests
+  def create
+    template = SupportRequestTemplate.friendly.find params[:template_id]
+
+    data = params[:data]
+    if data.empty?
+      render_error "Missing 'data' field", :unprocessable_entity
+    end
+
+    formatted_result = SupportRequestFormatterService.format template, data, current_user
+
+    repo_url = template.git_hub_repo
+    title = formatted_result.title
+    body = formatted_result.body
+
+    begin
+      url = git_hub_agent_service.create_issue repo_url, title, body
+    rescue => e
```

Comment:
No specific exceptions come to mind here; this is purely to catch all exceptions (except for things like ruby runtime errors) so that we can send a fallback error message to the user that some unknown error has occurred.